nipype/interfaces/io.py

```diff
@@ -2863,3 +2864,31 @@ def _list_outputs(self):
 
     def _add_output_traits(self, base):
         return add_traits(base, list(self.inputs.output_query.keys()))
+
+
+class ExportFileInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='Input file name')
+    out_file = File(mandatory=True, desc='Output file name')
+    check_extension = traits.Bool(True, desc='Ensure that the input and output file extensions match')
+    clobber = traits.Bool(desc='Permit overwriting existing files')
+
+
+class ExportFileOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='Output file name')
+
+
+class ExportFile(SimpleInterface):
```

Comment:
Could you make a docstring that explains what you're doing here and gives an example or two? The other interfaces in this file are good examples, if a little inconsistently formatted. I would go with the formatting in:

https://github.com/nipy/nipype/blob/47bb8e8376d1847fc377d41ddf82a1911227ed79/nipype/interfaces/io.py#L1297-L1327

I'm sorry to draw this process out... I'm not sure why I didn't bring this up before.